docs/integration-test-helpers/masonry/MasonryContainer.js

```diff
@@ -304,14 +366,16 @@ export default class MasonryContainer extends Component<Props, State> {
     }
 
     if (noScroll) {
-      dynamicGridProps.scrollContainer = () => undefined;
+      dynamicGridProps.scrollContainer = undefined;
     } else if (hasScrollContainer) {
-      dynamicGridProps.scrollContainer = () =>
-        typeof document === 'undefined'
-          ? undefined
-          : document.querySelector('[data-scroll-container]');
+      const query = document.querySelector('[data-scroll-container]');
+      if (typeof document === 'undefined' || query === null) {
+        dynamicGridProps.scrollContainer = undefined;
```

Comment:
This is a runtime change

Reply:
Same as  https://github.com/pinterest/gestalt/pull/2867#discussion_r1177215478